packages/langium/src/generator/node-processor.ts

```diff
@@ -70,17 +58,15 @@ class Context {
     }
 }
 
-export function process(node: GeneratorNode, defaultIndentation?: string | number): string {
+export function processNode(node: GeneratorNode, defaultIndentation?: string | number): string {
```

Comment:
Maybe even more specific, like `processGeneratorNode`? We need to consider that currently we export everything into the index, so users import it from `'langium'`.